data-access-analysis-tools/adql-recipes.rst

```diff
@@ -74,6 +74,7 @@ Exploring tables
 When learning about the contents of a table, it can be handy to simply retrieve all columns for "a bunch" (hundreds to thousands) of rows
 and take a look at the results.
 For this use-case, it is recommended to use the ``SELECT TOP`` statement, like in the example below that just retrieves the first 100 rows of the ``Object`` table.
+To directly go to the query results page, click this `link <https://data.lsst.cloud/portal/app/?api=tap&service=https://data.lsst.cloud/api/tap&adql=SELECT%20TOP%20100%20*%20FROM%20dp02_dc2_catalogs.Object&execute=true>`__.
```

Comment:
One of the recommendations for improving accessibility for screen readers is to use more descriptive names, e.g., instead of "click this link", something like "Go to the query results page", as the text for the link (as in the example in the following page: https://iota-school.github.io/notebooks-for-all/exports/resources/event-hackathon/accessibility-tips-for-jupyter-notebooks/#descriptive-text-for-visual-areas)

Reply:
Same comment for the resto of changes below.

Reply:
Thanks for the suggestion. It's incorporated.